integrations/git_test.go

```diff
@@ -159,10 +159,11 @@ func lfsCommitAndPushTest(t *testing.T, dstPath string) (littleLFS, bigLFS strin
 			t.Skip()
 			return
 		}
+		var err error
```

Comment:
```suggestion
```

Reply:
No, it doesn't work, the change is by purpose.

Because the return type of `RunWithXxx` is `RunError`, not `error`

Reply:
More details: why using `RunError` but not `error`: it reminds the developers that such error contains the `Stderr`, it's different from a normal command error.

Reply:
I can remove some unnecessary `var err error`, just a minute.

Reply:
All unnecessray `var err error` are removed. The remaining (about 4~5) are still necessary. I think it won't cause any misunderstanding now.